test/EFCore.PG.FunctionalTests/Query/GuidNpgsqlTest.cs

```diff
@@ -0,0 +1,117 @@
+using System;
+using System.Linq;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.TestUtilities;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Npgsql.EntityFrameworkCore.PostgreSQL.TestUtilities;
+using Xunit;
+
+namespace Npgsql.EntityFrameworkCore.PostgreSQL.Query
+{
+    public class GuidNpgsqlTest : IClassFixture<GuidNpgsqlTest.GuidFixture>
```

Comment:
These tests are great, but maybe we should include them in `SimpleQueryNpgsqlTest.Where.cs` (in a region) as with other similar translation tests? This would avoid the explosion of files and fixtures for each translator etc.

Reply:
Good call. It looks like the tests for SqlServer are in `SimpleQueryTestBase.Functions.cs`, so I moved these into `SimpleQueryNpgsqlTest.Functions.cs`.